.gitignore

```diff
@@ -12,3 +12,4 @@
 coverage_report/
 .coverage
 *.egg-info/
+.*.sw[a-z]
```

Comment:
If this is for vim swap files, let's remove it from here and you can put it in your global gitignore. Info about this here: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
oh ... thanks. i will include that next push